lib/index-strict.js

```diff
@@ -3,6 +3,9 @@
 var Plotly = require('./core');
 
 Plotly.register([
+    require('./splom-strict'),
+    require('./scattergl-strict'),
+    require('./parcoords-strict'),
```

Comment:
The order of registering these traces are not the same as main index.

Reply:
They are out of order because the index files do not contain them and they are inserted into the index-strict copy during our pre-bundling.

Are there any side effects from registration order that I might have missed?

Reply:
It would be safer to possibly keep the order as the main index. Wondering if you could replace `/scattergl` with `/scattergl-strict` in place?